src/main/java/Date.java

```diff
@@ -0,0 +1,42 @@
+import java.text.SimpleDateFormat;
+
+/**
+ * Date representation of date string that's given
+ */
+public class Date {
+    boolean exists = false;
+    String precursor = "";
+    SimpleDateFormat formatter = new SimpleDateFormat("dd/MM/yy HHss");
+    java.util.Date data;
+    public Date(String dateString) {
+        precursor = dateString.split(" ", 2)[0];
+        if (!dateString.equals("  ")) {
+            try {
+                this.data = formatter.parse(dateString.split(" ", 2)[1]);
+                this.exists = true;
+            }
+            catch (Exception e) {
+                System.out.println("WrongDateFormat");
+            }
+        }
+    }
+
+    public String format() {
+        if (exists) {
+            return precursor + " " + formatter.format(data);
+        }
+        else {
+            return "NIL";
+        }
+    }
+
+    @Override
+    public String toString() {
+        if (exists) {
+            return "(" + precursor + " " + formatter.format(data) + ")";
+        }
```

Comment:
Try to conform to the 2103 coding style!
the else { should be on the same line as '}'